CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## [0.0.8]
 
+### Removed
+* Removed mentions of Dash events in OncoPrint component.
+
 ### Added
 * Added ability to define custom colours in styles parser for Molecule3D. 
 
+
```

Comment:
I guess we don't want this extra blank line.

Reply:
Good catch!